app/data_providers/user_recommendation_preferences_provider.py

```diff
@@ -8,12 +8,18 @@
 from app import config
 from app.data_providers.topic_provider import TopicProvider
 from app.models.topic import TopicModel
-from app.models.user_recommendation_preferences import UserRecommendationPreferencesModel
+from app.models.user_recommendation_preferences import UserRecommendationPreferencesModel, \
+    UserRecommendationPreferencesModelV2
 
 
 class UserRecommendationPreferencesProvider:
+    """
+    Put or fetch user preferred topics in a Feature Group keyed on the integer user id.
+    """
+
     _FEATURE_GROUP_VERSION = 1
     _FEATURE_NAMES: List[str] = ['user_id', 'updated_at', 'preferred_topics']
+    _FEATURE_ID_NAME: str = 'user_id'
```

Comment:
I see that you have used the [hashed_user_id](https://github.com/Pocket/recommendation-api/pull/942/files#diff-e2ad3ff4166c3a19b996b8d510e6469502daf99882db0dbcd2acdff621fbf0a5R131)  for the _str_ UserID elsewhere. Is there a reason you want to use `user_id` here?

Reply:
This is the v1 feature store that is keyed on the integer user_id.

Reply:
> This is the v1 feature store that is keyed on the integer user_id.

For Feature groups, are we using `str` for all fields (including the integer data types)?

Reply:
Yes